tests/screenshot/apps/components/Alert.js

```diff
@@ -11,12 +11,15 @@ import img from '../../images/300x300.png';
 
 import {withConfig, withProps, LoremString} from './utils';
 
+import css from './Alert.module.less';
 
 // Only type: 'fullscreen' supports title prop
 const fullscreenTests = [
 	<Alert open title="Title" />,
 	<Alert open>Alert!</Alert>,
-	<Alert open>{LoremString}</Alert>
+	<Alert open>{LoremString}</Alert>,
+	<Alert open>{LoremString}</Alert>,
```

Comment:
Please remove the line
```suggestion
	<Alert open>{LoremString}</Alert>,
```

Reply:
ok